app/src/main/java/app/attestation/auditor/AttestationProtocol.java

```diff
@@ -247,6 +248,11 @@
                     new DeviceInfo(R.string.device_pixel_3, 3, 3, false /* uses new API */, true))
             .put("60D551860CC7FD32A9DC65FB3BCEB87A5E5C1F88928026F454A234D69B385580",
                     new DeviceInfo(R.string.device_pixel_3_xl, 3, 3, false /* uses new API */, true))
+            // CalyxOS - Same keys, need to figure out a way to differentiate.
+            .put("BCEBF6844F6B0FA2ABE8E62A9D0D57A324D0C02CEFDFA019FD49832F9ED39105",
+                    new DeviceInfo(R.string.device_pixel_2_generic, 2, 3, true, true))
+            //.put("BCEBF6844F6B0FA2ABE8E62A9D0D57A324D0C02CEFDFA019FD49832F9ED39105",
+            //      new DeviceInfo(R.string.device_pixel_3_generic, 3, 3, false /* uses new API */, true))
```

Comment:
Need to figure out a way to differentiate between these, as there's a difference in the rollback resistance implementation.

Any suggestions?

Reply:
You could just set it to false for now to make it work on both for now. I haven't implemented the check for the Pixel 3 yet because when I last checked the attestation information it wasn't properly setting the new rollback resistance flag. It's not a super important sanity check. Once the new way is supported on the Pixel 3, it will be possible to set it to true there.

Reply:
I don't even understand why they changed how it works. Look at KM_TAG_ROLLBACK_RESISTANCE vs. KM_TAG_ROLLBACK_RESISTANT in AuthorizationList.java. IIRC, neither was currently set for the Pixel 3, even though it has key rollback resistance. I haven't checked for a while. I think it was supposed to move to the new API but... they forgot?

Reply:
> You could just set it to false for now to make it work on both for now. I haven't implemented the check for the Pixel 3 yet because when I last checked the attestation information it wasn't properly setting the new rollback resistance flag. It's not a super important sanity check. Once the new way is supported on the Pixel 3, it will be possible to set it to true there.

Ok will do that for now, we aren't updating the rollback index now either way.

Reply:
It's a different thing than the rollback index. It's not important to have the sanity check for key rollback resistance being present though.